web-console/developer/advanced/advanced.md

```diff
@@ -12,6 +12,49 @@ When clicking on a specific project, Projects remains selected in the navigation
 
 ![Add project details](img/image2.png)
 
+## Project Access
+The Project Access page allows users to add or remove access to a project.
+
+![project access page](img/membership1.png)
+
+### Add and Remove Values
+* Clicking add access will add a row at the bottom of the list.
+* Clicking the remove icon on the right side of a row will remove the row.
+* After adding or removing access, users must click save at the bottom of the page to apply changes.
+* In the first section, users will be prompted to enter a name for the user.
+* In the second section, there are three options will chose from for selecting a role: admin, view, or edit.
+
+![add project access](img/membership3.png)
```

Comment:
This would be an interesting pattern to consider within the OpenShift Design Conventions. I'm not sure I've seen this inline informational message when a value has changed calling attention to the user to save the changes. I like it! I guess it also brings up the question of going to a page automatically in edit mode though... @rileyhuston any conventions here?

Reply:
Hey @lizsurette this is one of the things as we identified as a possible design convention!  It was actually initially used in the Environments tab before we had multiple perspectives.  We are now using it in the Parameters & Resources tab of the Pipeline page in addition to here in the Project Access page.  Sounds like they are also componentizing the code to make it more consistent, so it would be great to add to our Design Convention library.  Do we have a process to get things in there yet?

Reply:
Great! I'll let @rileyhuston speak to the process of adding into to the conventions "list to be discussed" when he's back from PTO :)